mdbook-course/src/course.rs

```diff
@@ -313,24 +307,18 @@ impl Session {
 
     /// Generate a Markdown outline for this session, for placement at the given
     /// path.
-    pub fn outline(&self, at_source_path: impl AsRef<Path>) -> String {
-        let mut outline = String::from("In this session:\n");
+    pub fn outline(&self) -> String {
+        let mut segments = Table::new(["Segment".into(), "Time".into()]);
```

Comment:
Perhaps "duration" is better here?

```suggestion
        let mut segments = Table::new(["Segment".into(), "Duration".into()]);
```